packages/rum-core/src/domain/rumEventsCollection/view/trackViews.ts

```diff
@@ -34,6 +39,8 @@ export interface ViewEvent {
   loadingType: ViewLoadingType
   cumulativeLayoutShift?: number
   hasReplay: boolean
+  inForeground?: boolean
```

Comment:
Do we need the foreground info on view event?

Reply:
No it's only used internally, might be able to remove it if you feel like it's misleading.
It's not exposed in `viewCollections`.

Reply:
if we don't need it, let's not add it

Reply:
Actually removed it as it makes no sense